docs-source/content/userguide/managing-domains/domain-resource.md

```diff
@@ -86,8 +86,9 @@ Elements related to domain [startup and shutdown]({{< relref "/userguide/managin
 
 Elements related to overriding WebLogic domain configuration:
 
-* `configOverrides`: The name of the config map for optional WebLogic configuration overrides.
-* `configOverrideSecrets`: A list of names of the secrets for optional WebLogic configuration overrides.
+These elements are under `configuration`.
+* `overridesConfigMap`: The name of the config map for optional WebLogic configuration overrides. The value should be empty/unset, if the `domainHomeSourceType` is set to `FromModel`.
+* `secrets`: A list of names of the secrets for optional WebLogic configuration overrides or model.
 
```

Comment:
This doc discusses the domain resource in general. Feels like we need to over the new elements in the latest schema here, for example, things under configuration.model and configuration.opss. Thoughts? @rjeberhard @tbarnes-us @jshum2479.

Reply:
For the MII specific ones, we could indicate that and point to the model-in-image doc.

Reply:
@doxiao Good catch.  I've updated the doc.  Please take a look at resolve this comment if you think it's OK.

Reply:
The update looks great.